airbyte-integrations/connectors/source-google-ads-asaas/acceptance-test-config.yml

```
@@ -1,6 +1,6 @@
 # See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
-connector_image: airbyte/source-google-ads-asaas:dev
+connector_image: airbyte/source-google-ads:dev
```

```suggestion
connector_image: airbyte/source-google-ads-asaas:dev
```
Manter o nome da imagem.